spec/type_aliases/fqdn_spec.rb

```diff
@@ -22,6 +22,7 @@
         "\nexample",
         "\nexample\n",
         "example\n",
+        '10.10.10.10.10',
```

Comment:
been somewhat pedantic this is a valid DNS name but its not a valid IANA DNS name.  i.e. someone could create there own private TLD for '10.'